scripts/run-migrations.ts

```diff
@@ -23,7 +23,12 @@ class MigrationRunner {
     
     try {
       await this.introspectProductionDatabase();
-      await this.generateMigrationDiff();
+      
+      if (this.isPreviewMode) {
+        await this.displayIntrospectionFiles();
+      } else {
+        await this.generateMigrationDiff();
+      }
```

Comment:
We shouldn't need to create a new method here. We should be able to reuse the existing logic downstream of `generateMigrationDiff` which eventually calls `displayMigrationFiles`